src/styles/product_card.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  display: block;
+  width: 200px;
+  padding: 32px 16px 17px;
+  background-color: var(--white-color);
+  border: 1px solid var(--border-grey-color);
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 45px;
+    
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+}
+
+.product__picture {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  padding: 0 3px;
+  height: 134px;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.product__name {
+  display: block;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: left;
+  color: var(--main-dark-color);
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.product__code {
+  font-weight: 400;
```

Comment:
it's a default value for this style
![image](https://user-images.githubusercontent.com/91411692/223100192-b6ebad0b-0f9e-4124-9d94-dbe49a8186d1.png)
![image](https://user-images.githubusercontent.com/91411692/223100211-e6c10509-09f2-4f56-8a43-eef96b99153d.png)

```suggestion
```